__tests__/groupController.test.js

```diff
@@ -0,0 +1,62 @@
+import 'babel-polyfill';
+import { createGroup, updateGroup, findAllGroups, deleteGroup } from '../src/hw-4/services/groupServices';
+import Group from '../src/hw-4/models/Group';
```

Comment:
Here and further - there is no `hw-4` folder in this branch, please fix the path.